packages/ember-glimmer/tests/integration/helpers/closure-action-test.js

```diff
@@ -246,46 +247,53 @@ moduleFor('Helpers test: closure {{action}}', class extends RenderingTest {
   }
 
   ['@test an error is triggered when bound action function is undefined']() {
-    let InnerComponent = Component.extend({
-    });
-
-    let OuterComponent = Component.extend({
+    this.registerComponent('inner-component', {
+      template: 'inner'
     });
-
-    this.registerComponent('inner-component', { ComponentClass: InnerComponent, template: 'inner' });
     this.registerComponent('outer-component', {
-      ComponentClass: OuterComponent,
       template: '{{inner-component submit=(action somethingThatIsUndefined)}}'
     });
 
-    // TODO: Improve this to test that the name of the parameter is contained
-    // within the error message.
+    // The assertion is different because in the HTMLBars case, the value is always a stream,
+    // whether or not the path read actually has value.
+    // In the Glimmer case, we're checking the value and not the reference, which could be undefined
+    // and thus throws the correct error.
     this.assert.throws(() => {
       this.render('{{outer-component}}');
-    }, /An action could not be made for `.*` in .*\. Please confirm that you are using either a quoted action name \(i\.e\. `\(action '.*'\)`\) or a function available in .*\./);
+    }, this.isGlimmer ?
```

Comment:
This is just a change in the assert though, is that worth the cost of one more test to maintain though